AppMap code objects sorting

moxw opened the pull request:
Fixes: #1420 

![Screenshot_20231103_032651](https://github.com/getappmap/appmap-js/assets/10359228/e52f2583-12ef-42fe-b510-ba4f87bb6888)
![Screenshot_20231103_032701](https://github.com/getappmap/appmap-js/assets/10359228/f884351b-6897-421a-8372-ab12fc12990e)

ahtrotta reviewed (changes requested):
Hi @moxw, this is looking great! I changed the CSS a little bit so that it's more clear to the user what is being used to sort. I see that you have tests for sorting by name and by occurrences, but can you add similar tests for sorting by execution order and elapsed time?

When you make changes, please use `fixup!` commits. To do this, make your changes and stage them, then `git commit --fixup=e0650699e`, where `e0650699e` is the commit hash. I'll squash the `fixup!` commits before merging. Thanks!

moxw commented:
> Hi @moxw, this is looking great! I changed the CSS a little bit so that it's more clear to the user what is being used to sort. I see that you have tests for sorting by name and by occurrences, but can you add similar tests for sorting by execution order and elapsed time?
> 
> When you make changes, please use `fixup!` commits. To do this, make your changes and stage them, then `git commit --fixup=e0650699e`, where `e0650699e` is the commit hash. I'll squash the `fixup!` commits before merging. Thanks!

Added the test cases.

appland-release commented:
:tada: This PR is included in version @appland/components-v3.13.0 :tada:

The release is available on:
- `@appland/components-v3.13.0`
- `@appland/components-v3.13.0`
- [GitHub release](https://github.com/getappmap/appmap-js/releases/tag/%40appland/components-v3.13.0)
- `@appland/components-v3.13.0`

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
